app/services/event_log/move_runner.rb

```diff
@@ -11,27 +11,29 @@ def call
       # iterate over all events in the log and apply changes to the move
       events.each do |event| # NB: do not use events.find_each as it will break the ordering
         case event.event_name
-        when Event::CANCEL
-          move.status = Move::MOVE_STATUS_CANCELLED
-          move.cancellation_reason = event.cancellation_reason
-          move.cancellation_reason_comment = event.cancellation_reason_comment
+        when Event::ACCEPT
+          move.status = Move::MOVE_STATUS_BOOKED
         when Event::APPROVE
           move.status = Move::MOVE_STATUS_REQUESTED
           move.date = event.date
-        when Event::ACCEPT
-          move.status = Move::MOVE_STATUS_BOOKED
-        when Event::REJECT
+        when Event::CANCEL
           move.status = Move::MOVE_STATUS_CANCELLED
-          move.rejection_reason = event.rejection_reason
-          move.cancellation_reason = Move::CANCELLATION_REASON_REJECTED
+          move.cancellation_reason = event.cancellation_reason
           move.cancellation_reason_comment = event.cancellation_reason_comment
-          move.rebook if event.rebook?
         when Event::COMPLETE
           move.status = Move::MOVE_STATUS_COMPLETED
         when Event::LOCKOUT
           # no action to perform when a move is locked out, this event is purely for auditing
         when Event::REDIRECT
           move.to_location = event.to_location
+        when Event::REJECT
+          move.status = Move::MOVE_STATUS_CANCELLED
+          move.rejection_reason = event.rejection_reason
+          move.cancellation_reason = Move::CANCELLATION_REASON_REJECTED
+          move.cancellation_reason_comment = event.cancellation_reason_comment
+          move.rebook if event.rebook?
```

Comment:
can we rebook outside the REJECT event? 
or `Event::REJECT` must be only inside this when-branch?

Reply:
hi @alexdesi rebooking a move is a feature only for the `reject` event